src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepoCommandTests.cs

```diff
@@ -107,6 +109,69 @@ public async Task Happy_Path()
+            const string adoTeamProject = "BlahTeamProject";
+            const string adoRepo = "foo-repo";
+            const string githubOrg = "foo-gh-org";
+            const string githubRepo = "gh-repo";
+            var adoRepoUrl = $"https://dev.azure.com/{adoOrg}/{adoTeamProject}/_git/{adoRepo}";
+            var adoToken = Guid.NewGuid().ToString();
+            var githubOrgId = Guid.NewGuid().ToString();
+            var migrationSourceId = Guid.NewGuid().ToString();
+            var migrationId = Guid.NewGuid().ToString();
+            var githubPat = Guid.NewGuid().ToString();
+            var githubRepos = new List<string> { githubRepo };
+
+            var mockGithub = new Mock<GithubApi>(null, null, null);
+            mockGithub.Setup(x => x.GetRepos(githubOrg).Result).Returns(githubRepos);
+
+            var mockGithubApiFactory = new Mock<GithubApiFactory>(null, null, null, null);
+            mockGithubApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>())).Returns(mockGithub.Object);
+
+            var environmentVariableProviderMock = new Mock<EnvironmentVariableProvider>(null);
+            environmentVariableProviderMock
+                .Setup(m => m.GithubPersonalAccessToken())
+                .Returns(githubPat);
+            environmentVariableProviderMock
+                .Setup(m => m.AdoPersonalAccessToken())
+                .Returns(adoToken);
+
+            var actualLogOutput = new List<string>();
+            var mockLogger = new Mock<OctoLogger>();
+            mockLogger.Setup(m => m.LogInformation(It.IsAny<string>())).Callback<string>(s => actualLogOutput.Add(s));
+            mockLogger.Setup(m => m.LogWarning(It.IsAny<string>())).Callback<string>(s => actualLogOutput.Add(s));
+
+            var expectedLogOutput = new List<string>
+            {
+                "Migrating Repo...",
+                $"ADO ORG: {adoOrg}",
+                $"ADO TEAM PROJECT: {adoTeamProject}",
+                $"ADO REPO: {adoRepo}",
+                $"GITHUB ORG: {githubOrg}",
+                $"GITHUB REPO: {githubRepo}",
+                $"The Org '{githubOrg}' already contains a repository with the name '{githubRepo}'. No operation will be performed"
+            };
+
+            // Act
+            var command = new MigrateRepoCommand(mockLogger.Object, mockGithubApiFactory.Object,
+                environmentVariableProviderMock.Object);
+            await command.Invoke(adoOrg, adoTeamProject, adoRepo, githubOrg, githubRepo, wait: false);
+
+            // Assert
+            mockGithub.Verify(m => m.GetRepos(githubOrg));
+
+            mockLogger.Verify(m => m.LogInformation(It.IsAny<string>()), Times.Exactly(6));
+            mockLogger.Verify(m => m.LogWarning(It.IsAny<string>()), Times.Exactly(1));
+            actualLogOutput.Should().Equal(expectedLogOutput);
+
+            mockGithub.VerifyNoOtherCalls();
+            mockLogger.VerifyNoOtherCalls();
```

Comment:
I wrote some comments for the other test to simplify the asserts, they apply here too.